config/electric.json

```diff
@@ -0,0 +1,84 @@
+{
+    "executable": "feelpp_toolbox_electric",
+    "output_directory": "/data/scratch/cladellash/feelppdb/toolboxes/electric",
```

Comment:
For toolboxes, isn't the output directory dependent on the `global_root` specified in `./feelppconfig` ? If so, it can cause trouble to use relative pathing. 
Also, I find absolute paths useful for applications that export huge amounts of data. For example, if the results need to be stored over NFS.